docs-ref-services/latest/app-configuration-readme.md

```diff
@@ -24,7 +22,7 @@ Use the client library for App Configuration to:
 [Source code](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/app-configuration/) |
 [Package (NPM)](https://www.npmjs.com/package/@azure/app-configuration) |
 [API reference documentation](https://docs.microsoft.com/javascript/api/@azure/app-configuration) |
-[Product documentation](https://docs.microsoft.com/en-us/azure/azure-app-configuration/) |
+[Product documentation](https://docs.microsoft.com/azure/azure-app-configuration/) |
```

Comment:
The readme update will be overwritten by the next release. I checked js repo. Readme content is correct over there. Please consider to update the readme in lang repo. 
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/app-configuration/README.md

Reply:
Same applies other changes.

Reply:
@sima-zhu Isnt the readme in the language repo already fixed to not have `en-us` in it? So, the changes in this PR should be safe